proto/tendermint/abci/types.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+package tendermint.abci;
+
+option go_package = "github.com/lazyledger/lazyledger-core/abci/types";
+
+// For more information on gogo.proto, see:
+// https://github.com/gogo/protobuf/blob/master/extensions.md
+import "google/protobuf/timestamp.proto";
+import "gogoproto/gogo.proto";
+
+// This file is copied from http://github.com/tendermint/abci
+// NOTE: When using custom types, mind the warnings.
+// https://github.com/gogo/protobuf/blob/master/custom_types.md#warnings-and-issues
+
+// Validator
+message Validator {
```

Comment:
Do we want to rename this to Aggregator? BlockProducer? 
Is there a well-established term in Rollups-land? 
cc @adlerjohn @musalbas

Reply:
My personal preference is block producer/proposer. There's no convention at this time unfortunately.

Reply:
An alternative, since "validator" has always been a poor word, would be "candidate" and "leader."

Reply:
I think we're a bit out-of-context here: Validator is used only in Evidence, and there is always one Validator (in Evidence). And this Evidence is ABCI type re-use here.
As it's only related to Evidence, 'Validator' seems a good name. In Header we use `proposer_address`, in commit there are just `signatures` without specifying is it validator/proposer/producer/whatever.

In my opinion there are two options:
1. Use ABCI Evidence as-is for now, reassess during fraud proofs work.
2. Create own Evidence type, and I would use "Proposer" instead of Validator.

Reply:
Yeah, let's keep the ABCI types unmodified. Internally, for types that do not cross the abci boundary, in optimint we can chose a different / better name.